docs/install.md

```diff
@@ -1,28 +1,47 @@
 #Installation and Settings
+##Prerequisites
 The MongoDB module and sub-modules need some configuration to be useful. This
 guide assumes that :
 
-* a [MongoDB][download] 3.0 to 3.6 instance is already installed, configured, and
+* a [MongoDB][download] 3.0 to 4.0 server instance is already installed, configured, and
   available to connect to from the Drupal instance.
 * the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
 * the [mongodb][mongodb] (not [mongo][mongo]) PHP extension version 1.1.7 or
   later is installed and configured.
 * PHP is version 7.[01].x. At this point, [PHP 7.2.x][php72] might not pass the
   test suite. It should be compatible by the time Drupal 8.6.0 is released.
 
+**NOTE** : There is a plan to support Drush 9.x and it can be tracked [here][drush9]
```

Comment:
Most lines in this paragraph are longer than 80 chars.

Reply:
Not sure what is the difference between the 2 choices you decribe ? The example on the sipper above lines 10->11 is OK for me: "PHP is version[...] pass the", and then the rest of the sentence is on the next line, indented: "    test suite[...]". Is it clear with that example ?

A few lines contain URLs longer than 80 chars themselves, but this doesn't count. This rule is for
text which can be wrapped.

Reply:
yes got it thanks.